### PR TITLE
Add DevTools extensions view as a tool window

### PR DESCRIPTION
**Pull request by helin24**
image

### CODE REVIEW

**Comment by helin24** [comment]
@kenzieschmoll was the background color issue fixed in standalone Chrome?

**Review by kenzieschmoll — COMMENTED** [review]
lgtm from DevTools perspective. But I'll defer to @jwren to review the plugin code.

I will try to repro the theme issue separately but that doesn't need to block this PR.

**Review by jwren — APPROVED** [review]

File: resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -641,6 +641,7 @@
 
     <toolWindow id="Flutter Deep Links" anchor="right" icon="FlutterIcons.Flutter_13" factoryClass="io.flutter.deeplinks.DeepLinksViewFactory" />
     <toolWindow id="Flutter DevTools" anchor="right" icon="FlutterIcons.Flutter_13" factoryClass="io.flutter.devtools.RemainingDevToolsViewFactory" />
+    <toolWindow id="Flutter Extensions" anchor="right" icon="FlutterIcons.Flutter_13" factoryClass="io.flutter.devtools.DevToolsExtensionsViewFactory" />
```
Comment (kenzieschmoll) [inline resources/META-INF/plugin.xml:644]: Whoops I missed this string. This should be "Flutter DevTools Extensions"

File: resources/META-INF/plugin_template.xml [hunk 2]
```diff
@@ -349,6 +349,7 @@
 
     <toolWindow id="Flutter Deep Links" anchor="right" icon="FlutterIcons.Flutter_13" factoryClass="io.flutter.deeplinks.DeepLinksViewFactory" />
     <toolWindow id="Flutter DevTools" anchor="right" icon="FlutterIcons.Flutter_13" factoryClass="io.flutter.devtools.RemainingDevToolsViewFactory" />
+    <toolWindow id="Flutter Extensions" anchor="right" icon="FlutterIcons.Flutter_13" factoryClass="io.flutter.devtools.DevToolsExtensionsViewFactory" />
```
Comment (kenzieschmoll) [inline resources/META-INF/plugin_template.xml:352]: same here

File: resources/META-INF/plugin.xml (hunk 1 above)
Comment (helin24) [reply resources/META-INF/plugin.xml:644]: oops fixed in new PR

**Pull request merged**